runtime/runtime-params-estimator/src/cost.rs

```diff
@@ -0,0 +1,140 @@
+use std::convert::TryFrom;
+use std::fmt;
+use std::str::FromStr;
+
+/// Kinds of things we measure in parameter estimator and charge for in runtime.
+///
+/// TODO: Deduplicate this enum with `ExtCosts` and `ActionCosts`.
+#[derive(Copy, Clone, PartialEq, Eq, Debug, PartialOrd, Ord)]
+#[repr(u8)]
+pub(crate) enum Cost {
```

Comment:
Guess we shall have stable string name for each cost.

Reply:
I don't think we need a super-stable name (we can change the name later), so just using enum name seems fine.